opentok/opentok.py

```diff
@@ -1365,3 +1367,66 @@ def __init__(
             timeout=timeout,
             app_version=app_version
         )
+
+    
+
+
+    def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> requests.Response:
+        """
+        Use this method so the moderator can mute all streams or a specific stream for OpenTok.
+        Please note that a client is able to unmute themselves.
+        This function stays in the OpenTok class and inherits from the Client class.
+
+        :param session_id gets the session id from another function called get_session()
+
+        :param stream_id gets the stream id from another function called get_stream(). Note
+        that this variable is set to an empty string in the function definition as a specific
+        stream may not be chosen.
+   
+        """
+
+        session_id = self.get_session()
+        stream_id = self.get_stream()
+        jwt = self._create_jwt_auth_header()
+
+        if not stream_id:
+            response = requests.post(
+            f'https://api.opentok.com/v2/project/{os.getenv("API_SECRET")}/{session_id}/stream/mute',
```

Comment:
Looks like 3.5 doesn't like the f syntax for strings, so need to switch it back

Reply:
Replaced the f syntax strings with compatible version

Reply:
Also, this is unrelated but the one thing I'm not doing in the new mute function is checking if the status code is valid and handling exceptions if it is not valid, so would I need to do something like this?

`if response status code is valid:
       return response
 elif the response status code == 403:
      raise exception 
 elif the response status code == 404:
     raise exception
  else:
      raise another exception`